kbase/narrative
# DATAUP-210 File size error dropzone

# Description of PR purpose/changes

This PR adds a file too large message as detailed in DATAUP-210. 
<img width="641" alt="Screen Shot 2020-10-27 at 6 03 13 PM" src="https://user-images.githubusercontent.com/56279459/97381841-c8476480-187e-11eb-9e5e-57885bc74ebe.png">

# Jira Ticket / Issue #
https://kbase-jira.atlassian.net/browse/DATAUP-210


# Testing Instructions
* Details for how to test the PR: 
* execute `make test-frontend-unit`
* Navigate to the data import tab on localhost. Add a file that exceeds 20 gb. You will now be able to see the error and click on the link to navigate to the globus docs or a globus account, depending on how you are signed in.

# Dev Checklist:


# Updating Version and Release Notes (if applicable)
